tutorials/3d/inverse_kinematics.rst

```diff
@@ -17,18 +17,18 @@ move each joint in our arm to get to that target.
 Initial problem
 ~~~~~~~~~~~~~~~
 
-Talking in Godot terminology, the task we want to solve here is to position
+In Godot parlance, the task we want to solve here is to position
 the 2 angles on the joints of our upperarm and lowerarm so that the tip of the
 lowerarm bone is as close to the target point (which is set by the target Vector3)
 as possible using only rotations. This task is calculation-intensive and never
 resolved by analytical equation solving, as it is an under-constrained
-problem which means that there is more than one solution to an
+problem, which means that there is more than one solution to an
 IK problem.
 
 .. image:: img/inverse_kinematics.png
 
 For easy calculation in this chapter, we consider the target being a
```

Comment:
I know you didn't touch this line yourself. But we should change it to, "we assume the target is a child of Skeleton".

Reply:
My bad, I seem to have commented below the wrong line. This change was intended for line 30. Which currently says "For easy calculation in this chapter, we consider the target being a child of Skeleton."

We should change it to "For easy calculation in this chapter, we assume the target is a child of Skeleton."

Reply:
Much obliged, @clayjohn.